health/micro-ui/web/micro-ui-internals/example/src/UICustomizations.js

```
@@ -859,7 +860,7 @@ export const UICustomizations = {
 
         case "MICROPLAN_STATUS":
           if (value && value != "NA") {
-            return t(value);
+            return <p>{t(Digit.Utils.locale.getTransformedLocale("MICROPLAN_STATUS_" + value))}</p>;
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)